pseudo.css

```diff
@@ -0,0 +1,13 @@
+.block {
+	display: inline;
+}
+
+.inline {
+	display: block;
+	border: 1px solid slategray;
+	margin: 25px 25px 25px 25px;
```

Comment:
Zoey might not have taught you yet, but you can replace margin: 25px 25px 25px 25px; With margin: 25px; This will assign 25px margin to all 4 sides.